cmd/aspect/test/test.go

```diff
@@ -7,6 +7,7 @@ Not licensed for re-use
 package test
 
 import (
+	"aspect.build/cli/pkg/pathutils"
```

Comment:
Please, move this import statement to the bottom. The convention is usually:

```
import (
<std packages>

<third-party packages>

<first-party packages>
)
```

Reply:
I'm thinking we should define a pre-commit hook for this - I don't know if there's some 3rd-party linter that does this?